client/telemetry/src/lib.rs

```diff
@@ -287,38 +277,98 @@ impl Stream for Telemetry {
 	}
 }
 
-impl slog::Drain for TelemetryDrain {
-	type Ok = ();
-	type Err = ();
-
-	fn log(&self, record: &slog::Record, values: &slog::OwnedKVList) -> Result<Self::Ok, Self::Err> {
-		let before = Instant::now();
+/// TODO doc
+#[derive(Debug, Default, Clone)]
+pub struct Telemetries {
+	senders: Senders,
+	node_pool: Arc<NodePool>,
+	wasm_external_transport: Option<wasm_ext::ExtTransport>,
+}
```

Comment:
@bkchr I'm starting to wonder if it wouldn't be good to have a static global variable for this? What do you think?

Reply:
Pro: I won't need to make a PR on polkadot because I won't change the API to create a substrate node anymore